repositories.yaml

```diff
@@ -1990,3 +1990,33 @@ applications:
     moz_pipeline_metadata_defaults:
       expiration_policy:
         delete_after_days: 400
+
+  - app_name: thunderbird_desktop
+    canonical_app_name: Thunderbird
+    app_description: Thunderbird email client
+    url: https://github.com/mozilla/releases-comm-central
```

Comment:
The github repo description says this repo is EXPERIMENTAL. Have we done everything we need to do to notify folks that we're placing a dependency on an their experimental repo? And what will happen should that repo go away or fall out of date?

Reply:
Yes, I asked for confirmation over slack that that's indeed the official mirror. I now asked for confirmation that, though marked as experimental, we can rely on it.